recipe/meta.yaml

```
@@ -24,9 +25,9 @@ requirements:
     - deprecated >=1.2.6
     - googleapis-common-protos >=1.52,<2.dev0
     - opentelemetry-api >=1.15,<2.dev0
-    - opentelemetry-proto ==1.26.0
-    - opentelemetry-sdk >=1.26.0,<1.27.dev0
-    - opentelemetry-exporter-otlp-proto-common ==1.26.0
+    - opentelemetry-proto ==1.30.0
+    - opentelemetry-sdk >=1.30.0,<1.31.dev0
+    - opentelemetry-exporter-otlp-proto-common ==1.30.0
     - requests >=2.7,<3.dev0
```

```suggestion
    - requests >=2.7,<3.0.dev0
```
or requests >=2.7,<3.0.0.dev0

---

I think `3.0.0.dev0` is more PEP-compliant than `3.0.dev0`. `3.0.dev0` might work for now but could stop working in the future without notice whereas `3.0.0.dev0` would require a [PEP 440](https://peps.python.org/pep-0440/) change to take effect.